command/harness/setup.go

```diff
@@ -184,6 +184,7 @@ func HandleSetup(
 		}
 		metrics.WaitDurationCount.WithLabelValues(r.PoolID, instance.OS, instance.Arch,
 			driver, metric.ConvertBool(fallback), strconv.FormatBool(poolManager.IsDistributed()), owner).Observe(setupTime.Seconds())
+		logr.WithField("setupTime", setupTime).Traceln("init time for vm setup")
```

Comment:
We don't have to print this on UI init logs. It should be part of dlite's internal logs only.